server/public/js/ops.js

```diff
@@ -0,0 +1,4 @@
+'use strict'
+
+const chapters = document.querySelector('.chapters');
+console.log(chapters);
```

Comment:
It's good to remove any console.logs leftover from debugging :)

Reply:
Is the `chapters` const being used anywhere? I see its not exported in the file.

Reply:
Removed. It was an option to loop over chapters to create chapter navigations, but I ended up resolving this in pug.